### PR TITLE
[IMP] l10n_rs: taxes

### PR DESCRIPTION
**Pull request by malb-odoo**
We want to invigorate a Tax Name Taxonomy so that tax names are as codified as possible. This allow for a better display on Invoice Description, and allowed us to implement a smart name_search.

In this PR, we change the taxes name so that it's more clear for the users

task-id: 3052677

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/l10n_rs/data/template/account.tax-rs.csv [hunk 1]
```diff
@@ -1,53 +1,53 @@
...
-"rs_sale_vat_20","10","","VAT 20%","20%","False","20.0","percent","sale","True","tax_group_vat_20","100","base","invoice","+003","","20%"
+"id","sequence","description","invoice_label","name","price_include","amount","amount_type","type_tax_use","active","tax_group_id","repartition_line_ids/factor_percent","repartition_line_ids/repartition_type","repartition_line_ids/document_type","repartition_line_ids/tag_ids","repartition_line_ids/account_id","description@sr@latin"
+"rs_sale_vat_20","10","VAT 20%","VAT 20%","20%","False","20.0","percent","sale","True","tax_group_vat_20","100","base","invoice","+003","","20%"
 "","","","","","","","","","","","100","tax","invoice","+103","rs_470",""
 "","","","","","","","","","","","100","base","refund","","",""
 "","","","","","","","","","","","100","tax","refund","-103","rs_470",""
-"rs_sale_vat_10","20","","VAT 10%","10%","False","10.0","percent","sale","True","tax_group_vat_10","100","base","invoice","+004","","10%"
+"rs_sale_vat_10","20","VAT 10%","VAT 10%","10%","False","10.0","percent","sale","True","tax_group_vat_10","100","base","invoice","+004","","10%"
```
Comment (oco-odoo) [inline addons/l10n_rs/data/template/account.tax-rs.csv:6]: Again, "VAT" equivalent missing in tranlsation

File: addons/l10n_rs/data/template/account.tax-rs.csv [hunk 2]
```diff
@@ -1,53 +1,53 @@
...
 "","","","","","","","","","","","100","tax","invoice","+103","rs_470",""
 "","","","","","","","","","","","100","base","refund","","",""
 "","","","","","","","","","","","100","tax","refund","-103","rs_470",""
-"rs_sale_vat_10","20","","VAT 10%","10%","False","10.0","percent","sale","True","tax_group_vat_10","100","base","invoice","+004","","10%"
+"rs_sale_vat_10","20","VAT 10%","VAT 10%","10%","False","10.0","percent","sale","True","tax_group_vat_10","100","base","invoice","+004","","10%"
 "","","","","","","","","","","","100","tax","invoice","+104","rs_471",""
 "","","","","","","","","","","","100","base","refund","","",""
 "","","","","","","","","","","","100","tax","refund","-104","rs_471",""
-"rs_sale_vat_0","30","","VAT 0%","0%","False","0.0","percent","sale","True","tax_group_vat_0","100","base","invoice","","",""
+"rs_sale_vat_0","30","VAT 0%","VAT 0%","0%","False","0.0","percent","sale","True","tax_group_vat_0","100","base","invoice","","",""
```
Comment (oco-odoo) [inline addons/l10n_rs/data/template/account.tax-rs.csv:10]: missing translation

File: addons/l10n_rs/data/template/account.tax-rs.csv [hunk 3]
```diff
@@ -1,53 +1,53 @@
...
+"rs_sale_vat_0_deduct_previous_tax","40","VAT 0% with the right to deduct previous tax","VAT 0% with the right to deduct previous tax","0% D","False","0.0","percent","sale","True","tax_group_vat_0","100","base","invoice","+001","","0% - odbitak prethodnog poreza"
 "","","","","","","","","","","","100","tax","invoice","","",""
 "","","","","","","","","","","","100","base","refund","","",""
 "","","","","","","","","","","","100","tax","refund","","",""
-"rs_sale_vat_0_no_deduct_previous_tax","50","","VAT 0% without the right to deduct previous tax","0% - prev. tax not deductible","False","0.0","percent","sale","True","tax_group_vat_0","100","base","invoice","+002","","0% - prethodni porez se ne odbija"
+"rs_sale_vat_0_no_deduct_previous_tax","50","VAT 0% without the right to deduct previous tax","VAT 0% without the right to deduct previous tax","0% ND","False","0.0","percent","sale","True","tax_group_vat_0","100","base","invoice","+002","","0% - prethodni porez se ne odbija"
```
Comment (oco-odoo) [inline addons/l10n_rs/data/template/account.tax-rs.csv:18]: translation seems inconsistent

File: addons/l10n_rs/data/template/account.tax-rs.csv [hunk 4]
```diff
@@ -1,53 +1,53 @@
...
 "","","","","","","","","","","","100","tax","invoice","","",""
 "","","","","","","","","","","","100","base","refund","","",""
 "","","","","","","","","","","","100","tax","refund","","",""
-"rs_purchase_vat_20","60","","VAT 20%","20%","False","20.0","percent","purchase","True","tax_group_vat_20","100","base","invoice","+008","","20%"
+"rs_purchase_vat_20","60","VAT 20%","VAT 20%","20%","False","20.0","percent","purchase","True","tax_group_vat_20","100","base","invoice","+008","","20%"
```
Comment (oco-odoo) [inline addons/l10n_rs/data/template/account.tax-rs.csv:22]: missing "VAT" in translation

File: addons/l10n_rs/data/template/account.tax-rs.csv [hunk 5]
```diff
@@ -1,53 +1,53 @@
...
 "","","","","","","","","","","","100","tax","invoice","+108","rs_270",""
 "","","","","","","","","","","","100","base","refund","","",""
 "","","","","","","","","","","","100","tax","refund","-108","rs_270",""
-"rs_purchase_vat_10","70","","VAT 10%","10%","False","10.0","percent","purchase","True","tax_group_vat_10","100","base","invoice","+008","","10%"
+"rs_purchase_vat_10","70","VAT 10%","VAT 10%","10%","False","10.0","percent","purchase","True","tax_group_vat_10","100","base","invoice","+008","","10%"
```
Comment (oco-odoo) [inline addons/l10n_rs/data/template/account.tax-rs.csv:26]: missing "VAT" in translation

File: addons/l10n_rs/data/template/account.tax-rs.csv [hunk 6]
```diff
@@ -1,53 +1,53 @@
...
 "","","","","","","","","","","","100","tax","invoice","+108","rs_271",""
 "","","","","","","","","","","","100","base","refund","","",""
 "","","","","","","","","","","","100","tax","refund","-108","rs_271",""
-"rs_purchase_vat_0","80","","VAT 0%","0%","False","0.0","percent","purchase","True","tax_group_vat_0","100","base","invoice","","",""
+"rs_purchase_vat_0","80","VAT 0%","VAT 0%","0%","False","0.0","percent","purchase","True","tax_group_vat_0","100","base","invoice","","",""
```
Comment (oco-odoo) [inline addons/l10n_rs/data/template/account.tax-rs.csv:30]: missing translation

File: addons/l10n_rs/data/template/account.tax-rs.csv [hunk 7]
```diff
@@ -1,53 +1,53 @@
...
 "","","","","","","","","","","","100","tax","invoice","","",""
 "","","","","","","","","","","","100","base","refund","","",""
 "","","","","","","","","","","","100","tax","refund","","",""
-"rs_purchase_import_vat_0","90","","VAT 0% - import VAT exempt","0% - import","False","0.0","percent","purchase","True","tax_group_vat_10","100","base","invoice","+006","","0% - uvoz"
+"rs_purchase_import_vat_0","90","VAT 0% - import VAT exempt","VAT 0% - import VAT exempt","0% EX","False","0.0","percent","purchase","True","tax_group_vat_10","100","base","invoice","+006","","0% - uvoz"
```
Comment (oco-odoo) [inline addons/l10n_rs/data/template/account.tax-rs.csv:34]: translation seems inconsistent

(on subsequent lines as well)

**Review by oco-odoo — CHANGES_REQUESTED** [review]

**Comment by malb-odoo** [comment]
Description column was wrong, it should be corrected and match the translation

**Comment by oco-odoo** [comment]
@robodoo r+

**Pull request closed without merging**